zanata-war/src/main/java/org/zanata/service/impl/ConfigurationServiceImpl.java

```diff
@@ -105,7 +105,6 @@ public String getConfig() {
                             .append("\n")
                             .append(makeProjectTypeSection(getProjectType()))
                             .append("\n");
-            appendLocalesIfPresent();
```

Comment:
But what about the names of the PO files (and the locale directories they
live in, inside zips)?  Will they still be consistent with the behaviour of
the cli client?  Will the user still be able to download an offline
translation zip, extract it, edit some po files and run zanata-cli push,
and have it all work?

On 25 February 2015 at 11:01, David Mason <notifications@github.com> wrote:

> In
> zanata-war/src/main/java/org/zanata/service/impl/ConfigurationServiceImpl.java
> <https://github.com/zanata/zanata-server/pull/651#discussion_r25309826>:
>
> > @@ -105,7 +105,6 @@ public String getConfig() {
> >                              .append("\n")
> >                              .append(makeProjectTypeSection(getProjectType()))
> >                              .append("\n");
> > -            appendLocalesIfPresent();
>
> @seanf <https://github.com/seanf> all generated zanata.xml files will no
> longer have a <locales> section. That includes the zanata.xml in zip
> downloads. It should not change anything about po files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/651/files#r25309826>.
>